dahyun1226/app/src/main/java/com/example/android_toy_project_study_2020_mvvm/api/RetrofitServiceInterface.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.android_toy_project_study_2020_mvvm.api
+
+import com.example.android_toy_project_study_2020_mvvm.data.GithubDetailRepoData
+import com.example.android_toy_project_study_2020_mvvm.data.GithubDetailUserData
+import com.example.android_toy_project_study_2020_mvvm.data.GithubResponseData
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+interface RetrofitServiceInterface {
+
+    @GET("/search/repositories")
+    fun requestGithubResponse(
```

Comment:
HTTP 프로토콜을 나타내는 GET, POST, PUT, DELETE 를 네이밍으로 많이 사용합니다.
참고로 request와 response라는 상반되는 단어를 같이 사용했는데 각 단어가 통신에서 어떻게 사용되는지 한번 더 고민해 보면 좋을 것 같습니다.

```suggestion
    fun getGithubRepositories()
```